components/gantry/oneaxis/oneaxis.go

```diff
@@ -203,11 +203,11 @@ func (g *oneAxis) homeTwoLimSwitch(ctx context.Context) error {
 		return err
 	}
 
-	g.logger.Debugf("positionA: %0.2f positionB: %0.2f", positionA, positionB)
-
 	g.positionLimits = []float64{positionA, positionB}
 	g.positionRange = positionB - positionA
 
+	g.logger.Infof("positionA: %0.2f positionB: %0.2f range: %0.2f", g.positionLimits[0], g.positionLimits[1], g.positionRange)
```

Comment:
I'd put this back to a debug log. It's not something anyone will care about unless you're trying to debug something with your gantry.

Reply:
Oh, this has already been merged. :shrug: I don't feel strongly, no need to change it in a follow-up PR until the logging gets so annoying that someone complains (which might never happen).